orc8r/cloud/go/obsidian/swagger/spec_servicer.go

```diff
@@ -46,6 +46,7 @@ func NewSpecServicer(spec string) protos.SwaggerSpecServer {
 	return &specServicer{spec}
 }
 
+// GetSpec returns the associated spec as a YAML string
```

Comment:
This comment is superfluous since the method is an impl of the protos.SwaggerSpecServer method. Only include this per-impl comment if it's documenting something idiosyncratic to the particular implementation

Reply:
Hmm I was thinking the same thing but you had a previous comment saying to add a comment for all public methods and it also seems like the Codacy Check requires it?